modules/n1ql/pages/n1ql-language-reference/infer.adoc

```diff
@@ -73,26 +75,60 @@ For example, one of the Couchbase training modules has music tracks with a sub-d
 While this pattern may not be ideal for a number of reasons, if your data follows this pattern it might seem that the data has a huge number of ‘fields’, since a data value is being used as a field name.
 When the schema inferencing process sees more than [.var]`dictionary_threshold` fields with different names, but the same sub-document schema, it collapses them into a single schema field marked as a dictionary.
 
-*RBAC Privileges*
+== Schema Output
+
+The statement returns the output in the http://json-schema.org/documentation.html[JSON Schema draft^] format as specified by http://json-schema.org/[json-schema.org^].
+It supports the following data types: array, boolean, integer, null, number, and object.
+
+At the top level, the output contains an array of schemas.
+Each schema recursively describes the structure of a flavor of document.
+For each identified attribute, the schema may contain the following details:
+
+Common Details::
+[.out]`#docs`;; Specifies the number of documents in the sample that contain this attribute.
+[.out]`%docs`;; Specifies the percentage of documents in the sample that contain this attribute.
+[.out]`samples`;; Contains sample values for the attribute found in the sample population.
+[.out]`type`;; Specifies specifying the identified data type of the attribute.
+
+Details for Array Data Type::
+[.out]`items`;; Contains details of the elements in the array.
+[.out]`minItems`;; Specifies the minimum number of elements (array size).
+[.out]`maxItems`;; Specifies the maximum number of elements (array size).
+
+Details for Object Data Type::
+[.out]`properties`;; Contains details of the properties of the object.
++
+Each property is described by a key-value pair, in which the key is the name of the property, and the value gives recursive details of that property.
+
+Details for Documents and Subdocuments::
+[.out]`$schema`;; Specifies the version of the JSON Schema standard.
+[.out]`Flavor`;; Specifies the flavor of a document or subdocument.
+
+== RBAC Privileges
 
-User executing the INFER statement must have the _Query Select_ privilege granted on the target keyspace/bucket.
+The user executing the INFER statement must have the _Query Select_ privilege granted on the target keyspace/bucket.
```

Comment:
Use second person, something like
To execute the INFER statement, you must have....

Reply:
Done